forge/src/main/java/hardcorequesting/forge/BookCatalystRecipeSerializer.java

```diff
@@ -6,12 +6,11 @@
 import net.minecraft.resources.ResourceLocation;
 import net.minecraft.world.item.crafting.RecipeSerializer;
 import net.minecraft.world.item.crafting.ShapedRecipe;
-import net.minecraftforge.registries.ForgeRegistryEntry;
 
 /**
  * Same as the corresponding class in the fabric module, but extends ForgeRegistryEntry
  */
-public class BookCatalystRecipeSerializer extends ForgeRegistryEntry<RecipeSerializer<?>> implements RecipeSerializer<BookCatalystRecipe> {
```

Comment:
With this change, this serializer should be equivalent to its fabric counterpart, and can be moved to the common project.